src/time-picker/tests/functional/TimePicker.ts

```diff
@@ -158,12 +159,6 @@ registerSuite('TimePicker', {
 					'The input value should not contain the disabled value.'
 				);
 			})
-			.end()
```

Comment:
The failing test was taken out as it does not test for the right thing. When a disabled menu item is clicked the correct behavior is that the menu should close.